### PR TITLE
Custom routes base stats

### LINKED ISSUE #1: Custom routes & removal of server stats
1. Allow a custom route for the health check to be passed as a parameter
2. Remove server-specific stats in favor of allowing client to pass in list

### PR DESCRIPTION
**Pull request by mrollinsiv**
Resolves #1

### CODE REVIEW

File: index.js [hunk 1]
```diff
@@ -1,14 +1,23 @@
...
   express = require('express'),
-  v8 = require('v8'),
   os = require('os'),
   fs = require('fs'),
-  yaml = require('js-yaml'),
-  healthRoute = process.env.HEALTH_CHECK_PATH;
+  yaml = require('js-yaml');
+
+let infoList = {
+  nodeVersionExpected: function () { return _get(getYaml('circle'), 'machine.node.version'); },
```
Comment (olsonpm) [inline index.js]: if we're not using circle ci then we can remove this and rename `nodeVersionActual` to just `nodeVersion`

Comment (olsonpm) [reply index.js]: and when we do that we can actually uninstall js-yaml and remove its related code (including the fs module and readFile)

File: index.js [hunk 2]
```diff
@@ -1,14 +1,23 @@
 'use strict';
 
-var statList;
-const _ = require('lodash'),
+const _assign = require('lodash/assign'),
```
Comment (olsonpm) [inline index.js]: looks like you're not using any of the lodash-specific features for _assign, _each or _map so we can use native js instead

File: index.js [hunk 3]
```diff
@@ -76,38 +77,42 @@ function renderHealth(allStats) {
...
-  router.get('/health-check', renderHealth(stats));
 
   return router;
 }
 
-module.exports = routes
-module.exports.renderHealth = renderHealth;
\ No newline at end of file
+module.exports = routes;
+module.exports.renderHealth = renderHealth;
```
Comment (olsonpm) [inline index.js]: it looks like renderHealth isn't really intended to be used by consumers.  Can we isolate this into its own module and not export it from index ?

That way we don't have to export it specifically for testing

File: index.js (hunk 1 above)
Comment (mrollinsiv) [reply index.js]: 👍

File: lib/render.js [hunk 4]
```diff
@@ -0,0 +1,38 @@
+const _isEmpty = require('lodash/isEmpty'),
```
Comment (olsonpm) [inline lib/render.js:3]: would be nice to add `'use strict';` at the top, but not that big a deal

**Review by olsonpm — APPROVED** [review]

File: lib/render.js (hunk 4 above)
Comment (mrollinsiv) [reply lib/render.js:3]: I updated eslint and fixed this.

**Pull request merged**